fuzzbench/run_experiment.py

```diff
@@ -25,6 +25,8 @@ def run_experiment():
     """Main experiment logic."""
     print('Initializing the job queue.')
     queue = rq.Queue()
+    # Initialization.
+    queue.empty()
```

Comment:
Why is this necessary? We create a fresh brand redis instance every time, no?

Reply:
No. If some jobs finished and the their results don't expired, we can still have those information in the queue, i.e., finished_job_registry.
For others, the same for unexpected termination of experiment.

Reply:
As mentioned above, when we e.g. `make run-experiment` or `make run-end-to-end-test`, we instantiate a new redis container (with docker compose), so there shouldn't be any existing data in it.

Reply:
unfortunately. no. So this is necessary.

Reply:
Could you try it on your side? For example, start `make run-end-to-end-test` and then stop it some time after.
Then starts it again.

Reply:
Right, if your redis is not empy, that means that the experiment run wasn't shut down properly (ie., the `docker-compose down` command didn't run). This will happen e.g., if the experiment was interrupted. Since the containers were not shut down, when you run docker compose up again, you'll just use the running containers. If you manually interrupt the experiment run, make sure you "docker compose down" before running again.

Reply:
Good suggestions!

Reply:
https://github.com/google/fuzzbench/pull/600 Done.